lib/experimental/block-comments.php

```diff
@@ -1,4 +1,18 @@
 <?php
+
+/**
+ * Adds support for block comments to the built-in post types.
+ *
+ * @return void
+ */
+if ( ! function_exists( 'gutenberg_block_comment_add_post_type_support' ) ) {
```

Comment:
There's no need for a `function_exists` check for the functions with `gutenberg_`.